sdcm/provision/scylla_yaml/node_builder.py

```diff
@@ -100,6 +114,10 @@ def broadcast_address(self) -> Optional[str]:
             return self._public_ip_address
         return None
 
+    @property
+    def enable_ipv6_dns_lookup(self) -> bool:
```

Comment:
Honestly I did not find how this property is used. It was in `sdcm.provision.scylla_yaml.cluster_builder.ScyllaYamlClusterAttrBuilder.enable_ipv6_dns_lookup`, I just moved it to node_builder, because it is defined on the node level now

Reply:
Found. If used here: [scylla_yaml.py](https://github.com/scylladb/scylla-cluster-tests/blob/0b080e2972b52d951cfa207adc364ab894d27d39/sdcm/provision/scylla_yaml/scylla_yaml.py#L291C3-L291C3)
It used in unit tests

Reply:
ok, now I get it. Thanks